corehq/ex-submodules/couchforms/getters.py

```diff
@@ -45,7 +45,7 @@ def get_instance_and_attachment(request):
         except MultiValueDictKeyError:
             raise MultipartFilenameError()
         else:
-            if instance_file.size > MAX_FORM_LENGTH:
+            if instance_file.size > settings.MAX_UPLOAD_SIZE:
                 logging.info("Domain {request.domain} attempted to submit a form exceeding 50MB")
```

Comment:
I know this isn't part of the PR, and so not blocking but I think this error message should reference the constant rather than a hard coded value, like the other ones

Reply:
sure @calellowitz, I don't mind fixing [your code](https://github.com/dimagi/commcare-hq/commit/100610b6614010462d933a33bc6878ef8a687716#diff-057bf3bb7bd65a86c8ea981fca24de73db91bd846818b13e8352e7a0ef56d825R49) 😛 https://github.com/dimagi/commcare-hq/pull/32549